hw/top_earlgrey/data/pins_cw310.xdc

```diff
@@ -79,8 +79,8 @@ set_property -dict { PACKAGE_PIN M21  IOSTANDARD LVCMOS33                   } [g
 set_property -dict { PACKAGE_PIN P18  IOSTANDARD LVCMOS33                   } [get_ports { IOC7  }]; #  P18:USRUSB_VBUS_DETECT
 set_property -dict { PACKAGE_PIN W21  IOSTANDARD LVCMOS33 PULLTYPE PULLDOWN } [get_ports { IOC8  }]; #  W21:USB_A18               TAP Strap 0         (SAM3X):A18
 set_property -dict { PACKAGE_PIN M22  IOSTANDARD LVCMOS33                   } [get_ports { IOC9  }]; #  M22:PMOD2_IO1             EarlGrey:GPIO       (BoB):PMOD1_CSB3
-set_property -dict { PACKAGE_PIN N19  IOSTANDARD LVCMOS33                   } [get_ports { IOC10 }]; #  N19:PMOD2_IO3             EarlGrey:GPIO       (BoB):PMOD1_CSB5
-set_property -dict { PACKAGE_PIN P26  IOSTANDARD LVCMOS33                   } [get_ports { IOC11 }]; #  P26:PMOD2_IO4             EarlGrey:GPIO       (BoB):PMOD1_CSB6
+set_property -dict { PACKAGE_PIN N19  IOSTANDARD LVCMOS33                   } [get_ports { IOC10 }]; #  N19:PMOD2_IO3             EarlGrey:I2C3_SCL   (BoB):PMOD1_CSB5
+set_property -dict { PACKAGE_PIN P26  IOSTANDARD LVCMOS33                   } [get_ports { IOC11 }]; #  P26:PMOD2_IO4             EarlGrey:I2C3_SDA   (BoB):PMOD1_CSB6
```

Comment:
For DV, that seems fine--It maps this instance to pins with "real" support for open-drain functionality.

The comment changes for the FPGAs seem fine too. It is a _slight_ deviation from our intention to represent a particular SKU, but we probably need to evolve beyond that sort of association anyway...

Reply:
I will say, though, that for hyperdebug platforms, the expectation would be that some board other than hyperdebug would actually exercise this I2C instance when connected to these pins. The Nucleo board doesn't offer I2C from the STM32L5 here.

You could argue that we shouldn't write this in the FPGA comments, then, since our "standard" test platform does not make this available.

For the SAM3X variant, you have hooked it up to a Pmod connector, so an external board is required anyway.

Reply:
Okay, so should I revert the change in `pins_cw310_hyperdebug.xdc` but keep the change in `pins_cw310.xdc`? What about `pins_cw341.xdc`?

Reply:
Yeah... I guess I've talked myself into siding with reverting the changes to `pins_cw310_hyperdebug.xdc`. We can keep the changes to `pins_cw310.xdc` and `pins_cw341.xdc`, if you like. Does that sound good to you?

I don't feel strongly here, though, so I won't complain either way, haha.